src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/consumer/impl/MessageConsumerImpl.java

```diff
@@ -58,6 +65,8 @@ public void readAndProcess() {
                 Map<String, Object> logMap = new HashMap<>();
                 logMap.put("Message Offset", msg.getOffset());
                 logMap.put("Deserialised message id", deserializedMsg.getMessageId());
+                logMap.put("Thread Name", Thread.currentThread().getName());
+                logMap.put("Class ID", id);
```

Comment:
instead of class ID have Message Consumer Id, as it's more of an id for the running message consumer (not kafka consumer)